packages/core/lib/modules/network-manager.js

```diff
@@ -27,13 +27,6 @@ export const SUBSTRATE_NETWORKS = {
 };
 
 function getNetworkInfo(networkId): NetworkInfo {
-  console.log('getNetworkInfo', networkId);
```

Comment:
I'm embarrassed that I missed that.

Reply:
Maybe in the future throw a TODO or something in when adding debug code

Reply:
Me too 😅